module/Application/view/application/index/index.phtml

```diff
@@ -4,7 +4,7 @@
     <p>
         Congratulations! You have successfully installed the
         <a href="https://github.com/zendframework/ZendSkeletonApplication" target="_blank">ZF Skeleton Application</a>.
-        You are currently running Zend Framework version <?= \Application\Module::VERSION ?>.
+        You are currently running Zend Framework version <?php echo \Application\Module::VERSION ?>.
```

Comment:
`<?=` is fine, as our target is 5.6+ (5.4+ has that *always* work, regardless of the `short_tags` configuration setting).